cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -87,7 +87,7 @@ data GenesisCreateTestNetDataCmdArgs = GenesisCreateTestNetDataCmdArgs
   { specShelley :: !(Maybe FilePath) -- ^ Path to the @genesis-shelley@ file to use. If unspecified, a default one will be used if omitted.
   , numGenesisKeys :: !Word -- ^ The number of genesis keys credentials to create and write to disk.
   , numPools :: !Word -- ^ The number of stake pools credentials to create and write to disk.
-  , numStakeDelegators :: !Word -- ^ The number of delegators to pools to create and write to disk.
+  , stakeDelegators :: !(StakeDelegatorsKind, Word) -- ^ The number of delegators to pools to create.
```

Comment:
Instead of a tuple what about:
```
data StakeDelegatorsKind 
  = OnDisk Word -- ^ Credentials are written to disk
  | Transient Word -- ^ Credentials are not written to disk
  deriving Show
```

Reply:
Sounds good to me, change done :+1: